build/prefix_log.go

```diff
@@ -98,13 +111,55 @@ func (p *PrefixLog) Critical(v ...interface{}) {
 	p.log.Critical(p.addArgsPrefix(v)...)
 }
 
+// TraceS writes a structured log with the given message and key-value pair
+// attributes with LevelTrace to the log.
+func (p *PrefixLog) TraceS(ctx context.Context, msg string, attrs ...any) {
```

Comment:
Can we create the context inside this method so the caller doesn't need to create it? Think this way we won't need more code lines when creating logs

Reply:
Passing in a context is part of what's cool about this feature cause we can add kv pair attributes to a context and that will get logged without us needing to remember to include it in the log every time